playbooks/roles/aws/tasks/main.yml

```diff
@@ -31,22 +31,22 @@
     - deploy
 
 - name: Set the MTU to 1500 temporarily
-  shell: /sbin/ifconfig eth0 mtu 1500 up
-  when: ansible_distribution in common_debian_variants
+  shell: "/sbin/ifconfig mtu {{ ansible_default_ipv4.interface }} 1500 up"
+  when: (ansible_distribution in common_debian_variants) and (ansible_default_ipv4.interface != 'ens3')
 
-- name: Check for eth0.cfg
+- name: Check for {{ ansible_default_ipv4.interface }}.cfg
   stat:
-    path: /etc/network/interfaces.d/eth0.cfg
-  register: eth0_cfg
+    path: "/etc/network/interfaces.d/{{ ansible_default_ipv4.interface }}.cfg"
+  register: ip_interface_cfg
   when: ansible_distribution in common_debian_variants
 
-- name: Set the MTU to 1500 inside eth0.cfg file
+- name: Set the MTU to 1500 inside {{ ansible_default_ipv4.interface }}.cfg file
   lineinfile:
-    dest: /etc/network/interfaces.d/eth0.cfg
-    regexp: "^post-up /sbin/ifconfig eth0 mtu 1500"
-    line: "post-up /sbin/ifconfig eth0 mtu 1500"
+    dest: "/etc/network/interfaces.d/{{ ansible_default_ipv4.interface }}.cfg"
+    regexp: "^post-up /sbin/ifconfig {{ ansible_default_ipv4.interface }} mtu 1500"
+    line: "post-up /sbin/ifconfig {{ ansible_default_ipv4.interface }} mtu 1500"
     insertafter: "^iface"
-  when: ansible_distribution in common_debian_variants and eth0_cfg.stat.exists
+  when: (ansible_distribution in common_debian_variants and ip_interface_cfg.stat.exists) and (ansible_default_ipv4.interface != 'ens3')
```

Comment:
@sidorovdmitry 
1)Jumbo frames are used for GE and more productive networks

2) some specific kwm interface configuration issue
```
sudo ifconfig mtu ens3 1500 up
SIOCSIFADDR: No such device
mtu: ERROR while getting interface flags: No such device
SIOCSIFADDR: No such device
mtu: ERROR while getting interface flags: No such device
mtu: ERROR while getting interface flags: No such device
```